react-app/src/component/Legend/ColorLegend.stories.tsx

```diff
@@ -59,9 +37,15 @@ const defaultProps = {
   legendScaleSize: 300,
 };
 
+export const DefaultStyle = Template.bind({});
+DefaultStyle.args = {
+  ...defaultProps,
+};
+
 export const TopLeftPosition = Template.bind({});
 TopLeftPosition.args = {
   ...defaultProps,
+  cssLegendStyles: { left: "0vw", top: "0vh" },
```

Comment:
Should this be imported from DEFAULT_STYLE in constants.ts?

Reply:
The point with the story is to demonstrate how to explicitly set the position